app/_landing_pages/konnect-platform/konnect-mcp.yaml

```
@@ -0,0 +1,468 @@
+metadata:
+  title: "{{site.konnect_product_name}} MCP Server"
+  content_type: landing_page
+  description: "Interact with Kong {{site.konnect_product_name}} through AI clients using MCP tools. Access gateway entities, debug API performance, and search documentation."
+  products:
+    - konnect
+  breadcrumbs:
+    - /konnect/
+    - /konnect-platform/kai/
+  tags:
+    - ai
+    - mcp
+  search_aliases:
+    - ai assistant
+  beta: true
+
+rows:
+  - header:
+      type: h1
+      text: "{{site.konnect_product_name}} MCP Server"
+      sub_text: "Interact with Kong {{site.konnect_product_name}} through AI assistants and IDE copilots using the Model Context Protocol (MCP). Access {{site.base_gateway}} entities, debug API performance, and search Kong documentation from your development environment."
+
+  - columns:
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: |
+                    The {{site.konnect_product_name}} MCP Server enables developers to manage API infrastructure and debug performance issues directly from their development environment. Query {{site.base_gateway}} entities, create debug sessions with active tracing, and search Kong documentation without leaving your IDE or terminal.
```

```suggestion
                    The {{site.konnect_product_name}} MCP Server enables developers to manage API infrastructure and debug performance issues directly from their development environment. Query {{site.base_gateway}} entities, create debug sessions with active tracing, and search Kong documentation without leaving your IDE or agentic CLI coding tool.
```

I suggest this to disambiguate with our `kongctl` tool which is a tool for terminal based resource management. The MCP server is typically driven by an agent process so I think this clarification is helpful.